medical_surgical_appointment/models/medical_surgical_appointment_practitioner.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class MedicalSurgicalAppointmentPractitioner(models.Model):
```

Comment:
This can be on medical_surgical_appointment, as this are lines of the appointment